config/coronavirus.uiowa.edu/user.role.editor.yml

```diff
@@ -73,8 +73,6 @@ permissions:
   - 'create uiowa_hero block content'
   - 'create uiowa_image_gallery block content'
   - 'create uiowa_slider block content'
-  - 'create uiowa_page_title_hero block content'
-  - 'create uiowa_spacer_separator block content'
```

Comment:
@bspeare Are you aware of any reason why we need to leave these permissions intact on coronavirus.uiowa.edu?